java/src/jmri/jmrix/roco/z21/simulator/Z21SimulatorAdapter.java

```diff
@@ -159,9 +159,14 @@ public void run() {
                 }
                 log.debug("Client Disconnect");
             }
-        } catch (SocketException | RuntimeException ex0 ) {
+        } catch (SocketException ex0 ) {
             log.error("Exception opening socket", ex0);
             return; // can't continue from this
+        } catch (RuntimeException rte) {
+            // sublcases of RuntimeException may occur at times other than 
```

Comment:
The word `sublcases` seems to be misspelled and should be `subclasses`.

Reply:
Thanks